babel.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
     plugins: [
+        '@babel/plugin-syntax-dynamic-import',
         '@babel/plugin-transform-react-jsx',
         'babel-plugin-preval',
-        'babel-plugin-dynamic-import-node',
```

Comment:
`babel-plugin-dynamic-import-node` makes dynamic imports impossible, it overrides `import` to `require`. It was intended to be used for server code (because Node needs require), but typescript handles this conversion from what I gather